lhotse/utils.py

```diff
@@ -66,6 +69,8 @@ def non_null_dict_factory(collection):
         for key, val in d.items():
             if val is None:
                 remove_keys.append(key)
+            if convert_named_tuples and isinstance(val, tuple) and hasattr(val, '_fields'):
+                d[key] = val._asdict()
```

Comment:
I think we should serialize named tuples (specifically, alignment) as lists rather dicts to save the space in the manifests.

Reply:
I have made this change in the latest commit. Some JSONL serialization tests are failing. I'm looking into that now.

Reply:
It seems this is because we convert the NamedTuple to a list, which now has 2 `float` types and an `str` type. The `pyarrow.json` cannot read lists with such mixed types. Do you have any suggestions to handle this?

Reply:
Got it. In that case I think what you suggested makes more sense than trying to hack it somehow. People using alignments in supervisions might want to use gzip compression.